docs/self-managed/zeebe-deployment/configuration/gateway.md

```diff
@@ -0,0 +1,256 @@
+| -------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| enabled              | Enables TLS authentication between clients and the gateway. This setting can also be overridden using the environment variable `ZEEBE_GATEWAY_SECURITY_ENABLED`. | False         |
+| certificateChainPath | Sets the path to the certificate chain file. This setting can also be overridden using the environment variable `ZEEBE_GATEWAY_SECURITY_CERTIFICATECHAINPATH`.   |               |
+| privateKeyPath       | Sets the path to the private key file location. This setting can also be overridden using the environment variable `ZEEBE_GATEWAY_SECURITY_PRIVATEKEYPATH`.      |               |
+
+#### YAML snippet
+
+```yaml
+security:
+  enabled: false
+  certificateChainPath:
+  privateKeyPath:
+```
+
+### zeebe.gateway.longPolling
+
+| Field   | Description                                                                                                                                      | Example Value |
+| ------- | ------------------------------------------------------------------------------------------------------------------------------------------------ | ------------- |
+| enabled | Enables long polling for available jobs. This setting can also be overridden using the environment `variable ZEEBE_GATEWAY_LONGPOLLING_ENABLED`. | True          |
+
+#### YAML snippet
+
+```yaml
+longPolling:
+  enabled: true
+```
+
+### zeebe.gateway.interceptors
+
+Consider reading our documentation on interceptors first.
+
+Each interceptor should be configured with the values described below:
+
+<table name="interceptors" id="interceptors">
+    <thead>
+        <tr>
+            <th>Field</th>
+            <th>Description</th>
+            <th>Example Value</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>id</td>
+            <td>Identifier for this interceptor. This setting can also be overridden using the environment variable `ZEEBE_GATEWAY_INTERCEPTORS_0_ID`.</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>jarPath</td>
+            <td>Path (relative or absolute) to the JAR file containing the interceptor class and its dependencies. All classes must be compiled for the same language version as Zeebe or lower. This setting can also be overridden using the environment variable `ZEEBE_GATEWAY_INTERCEPTORS_0_JARPATH`.</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>className</td>
+            <td>Entry point of the interceptor, a class which must:
+              <li>implement [io.grpc.ServerInterceptor](https://grpc.github.io/grpc-java/javadoc/io/grpc/ServerInterceptor.html)</li>
```

Comment:
this markdown link doesn't render for me, also needs to be a html link then? 😢

Reply:
fixed ✅